src/uoapi/course/course_info.py

```diff
@@ -92,17 +133,18 @@ def get_courses(link):
             title = re.sub(pt.code_re, '', title)
             title = re.sub(pt.credit_re, '', title).strip()
         try:
-            desc = course.find('p', attrs={'class':'courseblockdesc'})
+            desc = course.find('p', attrs={'class': 'courseblockdesc'})
             desc = desc.text.replace('\xa0', ' ').strip()
         except AttributeError as e:
             if desc is None:
                 desc = ''
             else:
                 print(course)
```

Comment:
This is also my bad here.
```suggestion
                print(course, file=sys.stderr)
```